src/main/kotlin/racingcar/communication/ControlTower.kt

```diff
@@ -1,12 +1,25 @@
 package racingcar.communication
 
-import racingcar.domain.racing.FinalLab
 import racingcar.communication.input.Input
 import racingcar.communication.output.Output
+import racingcar.domain.cars.RacingCar
+import racingcar.domain.racing.FinalLab
 
 object ControlTower {
     fun decideRacingLabs(input: Input, output: Output): FinalLab {
         output.finalLab()
         return FinalLab(value = input.finalLab())
     }
+
+    fun judgeTheWinningRecord(racingCarsList: List<RacingCar>): Int =
+        racingCarsList.maxOf { it.racingDistance() }
```

Comment:
```suggestion
        racingCarsList.maxOf(RacingCar::racingDistance)
```
전달되는 인자를 가공 없이 사용하는 경우에는 메서드 참조를 활용할 수 있어요

Reply:
아 이게 더 직관적일 수 있겠네요! 변경하겠습니다~